slave/cracker.go

```diff
@@ -0,0 +1,31 @@
+package slave
+
+import (
+	b64 "encoding/base64"
+	"fmt"
+
+	"github.com/aaronang/cong-the-ripper/lib"
+	"github.com/aaronang/cong-the-ripper/slave/brutedict"
+	"github.com/aaronang/cong-the-ripper/slave/hasher"
+)
+
+func Execute(task lib.Task, slave *Slave) {
+	bd := brutedict.New(&task)
+	digest, _ := b64.StdEncoding.DecodeString(task.Digest)
```

Comment:
Oh sorry, my mistake. But my point still stands doesn't it? If Digest in Job is a []byte, then you don't need to perform encoding or decoding anywhere in your code, except for testing.